lib/mixlib/log/logger.rb

```diff
@@ -0,0 +1,103 @@
+require "logger"
+
+# A subclass of Ruby's stdlib Logger with all the mutex and logrotation stuff
```

Comment:
wouldn't some customers be relying on the logrotation stuff?

Reply:
nevermind, i see the big block of red where this was copied from monologger in the other PR...

Reply:
> wouldn't some customers be relying on the logrotation stuff?

@lamont-granquist you should have trusted your instincts, yes at least one of your customers misses log rotation in Windows. 😢